doc/src/modules/solvers/solveset.rst

```diff
@@ -459,17 +461,19 @@ How do we deal with cases where only some of the solutions are known?
  equation we encounter in mathematics is an ideal case for solvers in
  a Computer Algebra System. We always have some cases, which are not
```

Comment:
When cases which are not solved or can only be solved incompletely, a `ConditionSet` is used and acts as an unevaluated solveset object.